app/services/tinyurl.rb

```diff
@@ -0,0 +1,6 @@
+class TinyURL
```

Comment:
Recommend changing the name of this class to something generic, such as `ShortenUrl`.  This is in case we need to change to another shortening service in the future.